### PR TITLE
fix: follow up with doc fixes and config path creation

### PR DESCRIPTION
**Pull request by evan-forbes**
## Description

quick follow up for the last tracing PR

### CODE REVIEW

File: pkg/trace/fileserver.go [hunk 1]
```diff
@@ -177,12 +176,9 @@ type S3Config struct {
 }
 
 // readS3Config reads an S3Config from a file in the given directory.
-func readS3Config(dir string) (S3Config, error) {
+func readS3Config(path string) (S3Config, error) {
 	cfg := S3Config{}
-	f, err := os.Open(filepath.Join(dir, "s3.json"))
```
Comment (evan-forbes) [inline pkg/trace/fileserver.go:182]: this bug required the file to be called s3.json

File: config/toml.go [hunk 2]
```diff
@@ -563,9 +563,10 @@ trace_type = "{{ .Instrumentation.TraceType }}"
 # The size of the batches that are sent to the database.
 trace_push_batch_size = {{ .Instrumentation.TraceBufferSize }}
 
-# The list of tables that are updated when tracing. All available tables and
-# their schema can be found in the pkg/trace/schema package. It is represented as a
-# comma separate string. For example: "consensus_round_state,mempool_tx".
+# The list of tables that are updated when tracing. Tables not included will
+# not be traced. All available tables and their schema can be found in the
+# pkg/trace/schema package. It is represented as a comma separate string.
+# For example: "consensus_round_state,mempool_tx".
```
Comment (evan-forbes) [inline config/toml.go:569]: happy to alter this more @staheri14

Comment (staheri14) [reply config/toml.go:569]: Thanks Evan, looks good!

**Review by staheri14 — APPROVED** [review]
LGTM!

**Review by rootulp — APPROVED** [review]

**Pull request merged**